visualization/app/codeCharta/codeCharta.service.spec.ts

```diff
@@ -29,7 +31,7 @@ describe("codeChartaService", () => {
 
 	describe("loadFiles", () => {
 		const expected: CCFile = {
-			fileMeta: { apiVersion: "1.1", fileName: "noFileName", projectName: "Sample Map" },
+			fileMeta: { apiVersion: "1.1", fileName: fileName, projectName: "Sample Map" },
```

Comment:
```suggestion
			fileMeta: { apiVersion: "1.1", fileName, projectName: "Sample Map" },
```

It might be good to use a linting rule to automatically use the shorthand notation. It is nicer to read and to grasp the code that way.